### DISCUSSION TITLE
Option to use "_" before wildcards

### DISCUSSION THREAD

**Discussion by samdakap** [simple-login/app#1501] [Feature Requests]
Hi, 
I am wondering if there would be an option to extend the list of characters between the root name and the wildcard?
instead of just "#, /, +", I would love to use "_".
Ideally, it could be a parameter in simplelogin.env.

**Comment by nguyenkims** [comment]
@samdakap a lot of characters aren't allowed in an email address and we have actually tried hard to come up with the 3 characters in directory :). The underscore can't be used here as it's already allowed in the alias prefix.
